lib/reporter.js

```diff
@@ -127,6 +127,34 @@ class AllureReporter extends events.EventEmitter {
                 allure.getCurrentSuite().testcases.pop()
             }
         })
+
+        this.on('allure:attachment', (attachment) => {
+            if (attachment.content == null) {
+                return
+            }
+
+            var allure = null
+            if (attachment.cid != null) {
+                allure = this.getAllure(attachment.cid)
+            } else if (attachment.test != null) {
+                allure = this.getAllureFromTestName(attachment.test)
```

Comment:
Do we actually need this? 
As I am following https://github.com/webdriverio/webdriverio/pull/1893, it likely will be resolved soon, and then we will always have capabilities id